### DISCUSSION TITLE
Old forum archive not available anymore?

### DISCUSSION THREAD

**Discussion by Mari-Val** [ImageMagick/ImageMagick#7746] [Kudos and Rants]
Hello, to all!

I had visited some topics on the old forum about spliting .ithmb files into separate images. Probably it would be this topic judging from the google search short description, but the page appears blank (mozilla and edge). No 404 just a white page loads.

https://imagemagick.org/discourse-server/viewtopic.php?t=24269

Is the old forum abandoned? Will it be come back online? (a similar question from 2020 was resolved, and i accessed the link in question in May 2024 but now something has broken i guess? )

imagemagick did an amazing work when i used it a few months back but i only did around 25% of my work and since then i lost the code. 

Everyday i am reminded that the old internet disappears and i have to self archive : P

**Comment by Enverex** [comment]
Almost every single search engine result being a link to a dead site is definitely a problem.

**Comment by fmw42** [comment]
I also get a blank image when trying to reach https://legacy.imagemagick.org/discourse-server/ and URLs within.

**Comment by dlemstra** [comment]
Probably some server issues. This might be resolved in a couple days.

**Comment by urban-warrior** [comment]
The ImageMagick discourse server has been replaced with Github issues and discussion forums.  The archived messages have been available for the last 10 years with band-aids and paper clips.  The last OS update broke it.  We may restore the service, if possible, or we may just retire it.  Until we get this issue resolved, we just redirect to the discussions forums.

**Reply by fmw42** [reply]
It has some good information and it would be a shame if that was disabled or not converted to the Github forum. Any chance of a conversion of the data esp in the Announcements area.

**Reply by urban-warrior** [reply]
It will take some time to see if we can restore the service.  When we have an update, we'll report back here.

**Reply by urban-warrior** [reply]
Service restored.

**Reply by Mari-Val** [reply]
Thank you immensely!

**Reply by SheepDomination** [reply]
I sorta like the older forum but hopefully the github discussion forum is just as effective.

**Reply by fmw42** [reply]
See https://legacy.imagemagick.org/discourse-server/

**Reply by Mustlane** [reply]
@urban-warrior 

Service seems to be down once again.

Also, just curious, but is there a way to make all of the data from /discourse-server (posts, threads etc.) public, so It could be hosted by others in case of yet another outage.

Thanks in advance for help!

**Comment by urban-warrior** [comment]
With the rise of AI technologies, ImageMagick's website has been overwhelmed by an ever-increasing number of web crawlers—often dozens per minute. This relentless spidering placed a significant strain on our servers, rendering the site progressively less usable for actual visitors.

In response, we implemented a variety of mitigation strategies to throttle the crawler traffic. While some measures offered limited relief, the only consistently effective solution was to temporarily take the legacy Discourse server offline during peak demand periods.

As a result, the main ImageMagick website now operates smoothly. However, the legacy Discourse server remains largely inaccessible, only coming online during brief windows when crawler activity subsides.

We continue to evaluate potential long-term solutions to this challenge, but an optimal and sustainable fix has yet to be identified.

**Reply by urban-warrior** [reply]
We tweaked the load settings, it should be back online now.

**Comment by fmw42** [comment]
It is working for me now.
